data/mods/CrazyCataclysm/crazy_speech.json

```diff
@@ -250,5 +250,45 @@
     "speaker": "aperture_potato",
     "sound": "Are you even listening to me?",
     "volume": 20
+  },
+  {
+	"type": "snippet",
+	"category": "clickbait",
+	"text": "KEVIN SACRIFICING GAMEPLAY FOR REALISM? THE SHOCKING TRUTH REVEALED"
+  },
+  {
+	"type": "snippet",
+	"category": "clickbait",
+	"text": "TEN FACTS ABOUT THE NEW BIONIC SYSTEM THAT WILL SHOCK YOU"
+  },
+  {
+	"type": "snippet",
+	"category": "clickbait",
+	"text": "YOU WON'T BELIEVE WHO JUST ADDED A NEW PROFESSION"
+  },
+  {
+	"type": "snippet",
+	"category": "clickbait",
+	"text": "TRIVIALIZE THE EARLY GAME WITH THIS ONE SIMPLE SPEAR WEAPON! CONTRIBUTORS HATE HIM"
```

Comment:
Whom?

Reply:
That's the secret. It's clickbait, so there *is* no actual person.

Reply:
No, I meant that your earlier version mentioned what this "HIM" did. But now I find this the two parts of the sentence are not related at all.

Reply:
This is a pretty common clickbait/joke title. The existence of the second sentence alludes to the fact that someone discovered said problematic thing, but the phrase still works even though they aren't named.